### DISCUSSION TITLE
osu!lazer randomly crash on Android device

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#20944] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

I want to record of footage me suffering in osu!lazer in Android phone and when I tried to load up replay it just crash.
This also happens sometimes randomly though any part of the game

### Screenshots or videos

https://user-images.githubusercontent.com/59832159/197773169-6172fda3-072b-47da-a63d-44235fb43919.mp4

### Version

![Screenshot_20221026-185800~2.png](https://user-images.githubusercontent.com/59832159/198020359-4c3b7b2c-86b6-4177-a803-0ad139182afc.png)

### Logs

I re entered the game again and osu!lazer resetted log and I can't really reproduce bug though I did find the logs in folder

**Comment by nagi-desuuu** [comment]
1. You can find the version code of the game in the settings and on the title screen.
2. Try enabling the option that lets you show hidden/system files. You can then retrieve the logs if you just go to `ppy.sh.osulazer/logs`
3. Can you give your Android version? It's related to question 1.

**Reply by timelessnesses** [reply]
It's android 11

**Reply by timelessnesses** [reply]
And the version says it's 0.1. 
![Screenshot_20221026-185359.png](https://user-images.githubusercontent.com/59832159/198019730-fb549131-77f5-45fc-bf70-b828ccdc59f7.png)

**Reply by timelessnesses** [reply]
welp found osu version

**Comment by Susko3** [comment]
A third party file manager (like [Total Commander](https://play.google.com/store/apps/details?id=com.ghisler.android.TotalCommander)) will allow you access the logs. The osu! verison is the one written at the bottom of the main osu! screen.

**Comment by timelessnesses** [comment, marked as answer]
I can't reproduce this now sorry.
